src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -22,8 +22,16 @@
 
   <ItemGroup>
     <PackageReference Include="LibLog" Version="5.0.6" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net461" Version="1.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)'=='net45' AND '$(TargetFramework)'=='net461'">
+    <DefineConstants>$(DefineConstants);NET_4</DefineConstants>
```

Comment:
What's `NET_4` for?

Reply:
Leftover from trying to serialize HttpRequestMessage